app/models/schedule.rb

```diff
@@ -9,4 +9,17 @@ class Schedule < ApplicationRecord
 
   validates :applies_from, presence: true
   validates :applies_to, presence: true
+
+  def self.default_for(course_group:, cohort: Cohort.current)
```

Comment:
@mooktakim any thoughts on this, yet to review your pr but just want to make sure this is aligned with you?